packages/core/test/bundling/forms_template_driven/BUILD.bazel

```
@@ -0,0 +1,85 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//tools:defaults.bzl", "jasmine_node_test", "ng_module", "ng_rollup_bundle", "ts_library")
+load("//tools/symbol-extractor:index.bzl", "js_expected_symbol_test")
+load("@npm//http-server:index.bzl", "http_server")
+
+ng_module(
+    name = "forms",
```

It might be worth keeping the convention of the other `bundling` projects and name this after the directory:

```suggestion
    name = "forms_template_driven",
```

(Same for `forms_reactive/BUILD.bazel`.)